main.js

```diff
@@ -0,0 +1,366 @@
+
+let startButton = document.getElementById("start");
+startButton.addEventListener("click", startGame);
+
+let singlePlayer = document.getElementById("single-player");
+singlePlayer.checked = true;
+singlePlayer.addEventListener("click", check);
+
+let twoPlayer = document.getElementById("two-player");
+twoPlayer.addEventListener("click", check);
+
+let statusBar = document.getElementById("status-bar");
+
+document.getElementById("audio").volume = 0.1;
+
+//////Global variable initializations////////////
+
+let timer = setInterval(countTimer, 1000);
+let totalSeconds = 0;
+let gameOver = true;
+let turnNumber = 0;
+let playerOne = "";
+let playerTwo = "";
+let xSquares = [];
+let oSquares = [];
+let emptySquares = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+let winCombos = [
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9],
+  [1, 4, 7],
+  [2, 5, 8],
+  [3, 6, 9],
+  [1, 5, 9],
+  [3, 5, 7]
+];
+
+//////Initializing game-board squares, should probably have made some loops but oh-well...////////
+
+let cell1 = document.getElementById("cell-1");
+let cell2 = document.getElementById("cell-2");
+let cell3 = document.getElementById("cell-3");
+let cell4 = document.getElementById("cell-4");
+let cell5 = document.getElementById("cell-5");
+let cell6 = document.getElementById("cell-6");
+let cell7 = document.getElementById("cell-7");
+let cell8 = document.getElementById("cell-8");
+let cell9 = document.getElementById("cell-9");
+
+cell1.squareNumber = 1;
+cell2.squareNumber = 2;
+cell3.squareNumber = 3;
+cell4.squareNumber = 4;
+cell5.squareNumber = 5;
+cell6.squareNumber = 6;
+cell7.squareNumber = 7;
+cell8.squareNumber = 8;
+cell9.squareNumber = 9;
+
+cell1.addEventListener("click", drawCell);
+cell2.addEventListener("click", drawCell);
+cell3.addEventListener("click", drawCell);
+cell4.addEventListener("click", drawCell);
+cell5.addEventListener("click", drawCell);
+cell6.addEventListener("click", drawCell);
+cell7.addEventListener("click", drawCell);
+cell8.addEventListener("click", drawCell);
+cell9.addEventListener("click", drawCell);
+
+let cellLookup = {
+  1: cell1,
+  2: cell2,
+  3: cell3,
+  4: cell4,
+  5: cell5,
+  6: cell6,
+  7: cell7,
+  8: cell8,
+  9: cell9
+};
+
+//////Defining functions used in the game/////////
+
+function randomInt(max, min) {
+  return Math.floor(min + Math.random() * (max - min + 1));
+}
+
+///////Game timer//////////////////////////////////////
+
+function countTimer() {
+  if (gameOver === false) {
+    ++totalSeconds;
+    let hour = Math.floor(totalSeconds / 3600);
+    let minute = Math.floor((totalSeconds - hour * 3600) / 60);
+    let seconds = totalSeconds - (hour * 3600 + minute * 60);
+    if (hour < 10) hour = "0" + hour;
+    if (minute < 10) minute = "0" + minute;
+    if (seconds < 10) seconds = "0" + seconds;
+    document.getElementById("timer").innerHTML =
+      "Time Elapsed: " + hour + ":" + minute + ":" + seconds;
+  }
+}
+
+////////Forces either Single Player or Two Player to be selected//////////
```

Comment:
Nice comments sectioning out functions and describing what the do. Also nice job keeping most of your functions relatively short and doing one action